modules/features/referrals/src/main/kotlin/au/com/shiftyjelly/pocketcasts/referrals/ReferralsClaimGuestPassViewModel.kt

```diff
@@ -166,6 +166,7 @@ class ReferralsClaimGuestPassViewModel @Inject constructor(
                 settings.referralClaimCode.set("", false)
                 (_state.value as? UiState.Loaded)
                     ?.let { loadedState -> _state.update { loadedState.copy(flowComplete = true) } }
+                job?.cancel()
```

Comment:
I realised [this commit](https://github.com/Automattic/pocket-casts-android/commit/00c3642483769f612510895a0c0828300ae9e6cf) did not cancel the job after flow was complete. I guess it got missed while rebasing a change.